terminus-ui/select/src/select.component.scss

```diff
@@ -1,3 +1,4 @@
+/* stylelint-disable */
```

Comment:
Same question as earlier about disabling for the whole file

Reply:
Fixed